src/main/java/com/openshift/internal/restclient/model/ImageStream.java

```diff
@@ -44,6 +45,16 @@ public ImageStream(ModelNode node, IClient client, Map<String, String []> proper
 		super(node, client, propertyKeys);
 		this.propertyKeys = propertyKeys;
 	}
+	
+	@Override
+	public DockerImageURI getDockerImageRepositorySpec() {
```

Comment:
was trying to explicitly distinguish it being in the spec section.  The other is actually depricated i blieve